components/wpress-browser.js

```diff
@@ -2,7 +2,10 @@ import path from 'path'
 import React from 'react'
 import cn from 'classnames'
 import { saveAs } from 'file-saver'
+import {Modal} from "./Modal";
```

Comment:
`{ Modal }` makes it slightly more readable

Reply:
@yani- can you fix this as well as the two lines below